### DISCUSSION TITLE
LSP snippes: support for `scope` and `isFileTemplate` fields

### DISCUSSION THREAD

**Discussion by hinell** [L3MON4D3/LuaSnip#845] [Q&A] [answered]
* [`"scope": ...`](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_language-snippet-scope) - Language scope per snippet
* [`"isFileTemplate": ....`](https://github.com/microsoft/vscode/issues/145929) - Snippet for empty files

Are these fields supported by Luasnip LSP engine withing `.code-snippets` files? Thanks. If not, should I fill a feature request?

**Comment by leiserfg** [comment]
Those are for `.code-snippets` files that whe don't support.

**Comment by hinell** [comment, marked as answer]
@leiserfg I'm sorry for duplicate. I've asked it already here: https://github.com/L3MON4D3/LuaSnip/issues/705#issuecomment-1419284976

Feel free to close this discussion or even delete it. Thank you!
